gloo/test/multiproc_test.h

```diff
@@ -64,18 +65,27 @@ class MultiProcWorker {
         new ::gloo::rendezvous::FileStore(storePath));
     semaphore_ = sem_open(semaphoreName.c_str(), 0);
     GLOO_ENFORCE_NE(semaphore_, (sem_t*)nullptr, strerror(errno));
+    cerf_file_path = std::string(std::tmpnam(nullptr)); // revert!
```

Comment:
`std::move`?

Reply:
I don't know how `std::move` might be applicable here, because the right-hand side is already `std::string&&`, so this line calls move assignment operator